[fmt:x64-windows] customized triplet build failure

**Host Environment**
- OS: [Windows 10]
- Compiler: vs2015+win10 SDK

**To Reproduce**
Steps to reproduce the behavior:
`vcpkg.exe install openimageio[opencolorio,pybind11,tools]:x64-windows`

**Failure logs** 
```
Starting package 68/88: fmt:x64-windows
Building package fmt[core]:x64-windows...
Could not locate cached archive: C:\Users\david\AppData\Local\vcpkg\archives\09\098304118bf6cc4d4193890032dede99aa44ba29.zip
-- Downloading https://github.com/fmtlib/fmt/archive/cd4af11efc9c622896a3e4cb599fa28668ca3d05.tar.gz... -- Extracting source D:/_DEV/VCPKG/downloads/fmtlib-fmt-cd4af11efc9c622896a3e4cb599fa28668ca3d05.tar.gz -- Applying patch fix-warning4189.patch
-- Using source at D:/_DEV/VCPKG/buildtrees/fmt/src/8668ca3d05-7f40cec0ee.clean
-- Configuring x64-windows
-- Building x64-windows-rel
-- Installing: D:/_DEV/VCPKG/packages/fmt_x64-windows/share/fmt/copyright
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-pkg-config-0.29.2-1-any.pkg.tar.xz...
-- Downloading https://repo.msys2.org/mingw/i686/mingw-w64-i686-libwinpthread-git-8.0.0.5906.c9a21571-1-any.pkg.tar.zst...
-- Fixing pkgconfig file: D:/_DEV/VCPKG/packages/fmt_x64-windows/lib/pkgconfig/fmt.pc
CMake Error at scripts/cmake/vcpkg_replace_string.cmake:11 (file):
  file failed to open for reading (No such file or directory):

    D:/_DEV/VCPKG/packages/fmt_x64-windows/share/fmt/fmt-targets-debug.cmake
Call Stack (most recent call first):
  ports/fmt/portfile.cmake:54 (vcpkg_replace_string)
  scripts/ports.cmake:79 (include)


Error: Building package fmt:x64-windows failed with: BUILD_FAILED
Please ensure you're using the latest portfiles with `.\vcpkg update`, then
submit an issue at https://github.com/Microsoft/vcpkg/issues including:
  Package: fmt:x64-windows
  Vcpkg version: 2020.06.15-nohash

Additionally, attach any relevant sections from the log files above.
```

**Additional context**
- I added `set(VCPKG_BUILD_TYPE release)` to x64-windows.cmake triplet

--- a/ports/fmt/CONTROL
+++ b/ports/fmt/CONTROL
@@ -1,4 +1,5 @@
 Source: fmt
 Version: 7.0.3
+Port-Version: 2
 Homepage: https://github.com/fmtlib/fmt
 Description: Formatting library for C++. It can be used as a safe alternative to printf or as a fast alternative to IOStreams.

--- a/ports/fmt/portfile.cmake
+++ b/ports/fmt/portfile.cmake
@@ -51,7 +51,7 @@ if(VCPKG_TARGET_IS_WINDOWS)
         )
     endif()
     if(NOT DEFINED VCPKG_BUILD_TYPE OR VCPKG_BUILD_TYPE STREQUAL "release")
-        vcpkg_replace_string(${CURRENT_PACKAGES_DIR}/share/fmt/fmt-targets-debug.cmake
+        vcpkg_replace_string(${CURRENT_PACKAGES_DIR}/share/fmt/fmt-targets-release.cmake
             "lib/fmt.dll"
             "bin/fmt.dll"
         )

